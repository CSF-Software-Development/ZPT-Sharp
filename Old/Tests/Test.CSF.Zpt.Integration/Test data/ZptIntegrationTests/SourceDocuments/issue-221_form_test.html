<html>
<head>
  <title>Issue 221 - HTML form tag</title>
</head>
<body tal:define="visible string:Yes">
<h1>HTML form element</h1>
<p>
  This test diagnoses <a href="https://github.com/csf-dev/ZPT-Sharp/issues/221">Issue #221</a>,
  a rendering bug for HTML <code>&lt;form&gt;</code> elements.
</p>
<form action="http://example.com/example" method="post" tal:condition="not:visible">
  <fieldset>
    <label>This should be hidden</label>
    <input type="text" />
    <button>Submit me please</button>
  </fieldset>
</form>
</body>
</html>
