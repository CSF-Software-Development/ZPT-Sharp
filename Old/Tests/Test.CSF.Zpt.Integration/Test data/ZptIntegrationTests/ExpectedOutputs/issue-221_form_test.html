<html>
<head>
  <title>Issue 221 - HTML form tag</title>
</head>
<body>
<h1>HTML form element</h1>
<p>
  This test diagnoses <a href="https://github.com/csf-dev/ZPT-Sharp/issues/221">Issue #221</a>,
  a rendering bug for HTML <code>&lt;form&gt;</code> elements.
</p>

</body>
</html>
